.github/workflows/dart.yml

```diff
@@ -0,0 +1,36 @@
+name: Dart
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      DISPLAY: ':99'
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: cedx/setup-dart@v2
```

Comment:
👍 - I'd hit this as well wrt chromedriver not being available in the dart container